votor-messages/src/slice_root.rs

```diff
@@ -0,0 +1,38 @@
+//! Module to define [`SliceRoot`].
+
+use {
+    serde::{Deserialize, Serialize},
+    solana_hash::Hash,
+    std::fmt::Display,
+};
+
+/// For every FEC set (AKA slice) of shreds, we have a Merkle tree over the shreds signed by the leader.
+/// This is the root of the tree.
+#[cfg_attr(
+    feature = "frozen-abi",
+    derive(AbiExample),
+    frozen_abi(digest = "7NWdx9UtHWLUeqzPF89smNiRta2uVaEEHZSBPxCtNuUe")
+)]
+#[derive(
+    Copy, Clone, Default, Debug, Hash, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize,
+)]
+#[repr(transparent)]
+pub struct SliceRoot(pub Hash);
```

Comment:
Why are we placing  it in Votor if this deals with shreds? Maybe time for rotor crate?

Also not sure why do we need a wrapper around Hash at all. Could we define an alias instead?

Reply:
Agreed with Alex (and Ashwin from https://github.com/anza-xyz/agave/pull/9130/files#r2560958890) - I don't think votor is the right place for this. Rather, I agree with Ashwin's comment from linked comment:

> Then the hash relevant to shreds could be named -> FecSetRoot and could live in ledger or turbine crates.

Since the erasure coding and FEC sets are created in `solana-ledger`, my vote would be `solana-ledger`